packages/element/src/serialize.js

```diff
@@ -445,6 +445,8 @@ export function renderElement( element, context = {} ) {
 				},
 				context
 			);
+		default:
+			throw new Error( tagName );
```

Comment:
I couldn' make it throw so I figured out we don't use the source code in all cases. It turned out it is even worse, we don't use in a majority of cases. I opened #8188 to fix it before I can continue debugging.